.changeset/old-students-taste.md

```diff
@@ -0,0 +1,6 @@
+---
+'@skeletonlabs/skeleton-svelte': minor
+---
+
+Replace deprecated $state.frozen with $state.raw
+Update Svelte dependency to the release version
```

Comment:
One more super quick request here - put this all in one line. Think of each changeset description as being a bullet list item on our changelog. Because that's exactly what it will be! :)